test/test-container-start-command.js

```diff
@@ -10,7 +10,7 @@ function container() {
 }
 
 tap.test('start default', function(t) {
-  var s = new container();
+  var s = newContainer();
```

Comment:
can you just rename newContainer to Container, then, since it is a constructor now? or container?